### ISSUE TITLE
steam: menu can't be selected by mouse, but can be selected by keyboard

### ISSUE THREAD

**Issue by hideyukn88** [microsoft/wslg#6]
# Environment

```none
Windows build number: 21340
Your Distribution version: Ubuntu
Your WSLg version: 0.2.13
```

# Steps to reproduce

Install steam
Launch steam
Open STORE (or any) drop down menu by mouse
Selected FEATURED (or any) and click.
The main window pane doesn't switch to selected contents.

![image](https://user-images.githubusercontent.com/13857962/112339920-52a0c280-8c7d-11eb-826f-99e37d9d1436.png)

#  Expected behavior

The main window pane switch to selected contents.

If menu item is selected by keyboard enter key, instead of mouse click, window correctly switch to selected contents.

**Comment by sundhaug92** [comment]
Confirming with 22000.194 and Ubuntu 20.04 x64

**Comment by isgallagher** [comment]
This is still an issue on Windows 11 (22526.1000).

**Comment by hideyukn88** [comment]
This issue is same root as https://github.com/microsoft/wslg/issues/141#issuecomment-1171495831, by removing `if` block mentioned in that comments, this will be fixed.

**Comment by hideyukn88** [comment]
Fixed in WSLg 1.0.41.

**Comment by ghostsquad** [comment]
Weird issue for me. I'm not running steam under WSL, but this is the first result that pops up for me when searching "can't click on menus on steam". I recently updated to Windows 11, and even after reinstalling Steam, I have a problem where, at some point (still unsure of the trigger), I'm not longer able to click on things in the upper (steam/view/friends & store/library/community & minimize/close)/lower (add game/downloads/friends & chat) "bars". I have to forcibly close Steam and reopen it.

Has anyone experienced this?
